vapi/library/library.go

```diff
@@ -212,6 +212,20 @@ func (c *Manager) PublishLibrary(ctx context.Context, library *Library, subscrip
 	return c.Do(ctx, url.Request(http.MethodPost, spec), nil)
 }
 
+// UpdateLibrary can update one or both of the tag Description and Name fields.
+func (c *Manager) UpdateLibrary(ctx context.Context, l *Library) error {
+	spec := struct {
```

Comment:
This is not going to work with the most recent VAPI version. Note the breaking change from the previous version you use: https://developer.vmware.com/docs/vsphere-automation/latest/content/api/content/library/library_id/patch/